### DISCUSSION TITLE
Map Tiles Look Blurry

### DISCUSSION THREAD

**Discussion by EULUSOY** [maplibre/maplibre-native#2874] [Q&A]
Hi,

Map tiles look blurry, and I'm using "tileSize": 256. As a temporary solution, I tried using "tileSize": 150, which makes the map look sharper, but the text becomes smaller. The map size of the XYZ map provided by the map provider is 256.

How can I adjust the sharpness without affecting the text size?

**Comment by louwers** [comment]
Please provide platform, device, MapLibre version, style used and other details needed to understand the issue you are facing.

You are probably using raster tiles?
